revolv/project/tests/tests_general.py

```diff
@@ -298,6 +298,7 @@ def perform_valid_donation(self):
             HTTP_X_REQUESTED_WITH='XMLHttpRequest'
         )
 
+    @override_settings(ENABLE_PAYMENT_CHARGING=False)
```

Comment:
We should have a comment saying why this is necessary to override (i.e. that dummy card numbers will cause the paypal sdk stuff to fail)